### PR TITLE
switched property indexing from vector to scalar

### PR DESCRIPTION
**Pull request by elalish**
Fixes https://github.com/elalish/manifold/discussions/664#discussioncomment-8765880

The bindings probably still need some work...

### CODE REVIEW

File: bindings/c/meshIOc.cpp [hunk 1]
```diff
@@ -52,15 +52,10 @@ void manifold_material_set_metalness(ManifoldMaterial *mat, double metalness) {
   from_c(mat)->metalness = metalness;
 }
 
-void manifold_material_set_color(ManifoldMaterial *mat, ManifoldVec4 color) {
+void manifold_material_set_color(ManifoldMaterial *mat, ManifoldVec3 color) {
```
Comment (geoffder) [inline bindings/c/meshIOc.cpp:55]: Just taking a look on my phone from the hospital with a suddenly one month early (healthy) new born daughter, but this looks ok to me. The `from_c` overload should be doing its job here so nothing needs to change past the type.

**Review by geoffder — APPROVED** [review]

**Comment by geoffder** [comment]
Still trying to check the emails! Sorry I've still been AWOL, PhD is done, and now baby is born. I hope I can make space to bring back some OSS and extracurricular coding back into my life before too long, but life continues to provide new challenges.

File: bindings/c/meshIOc.cpp (hunk 1 above)
Comment (elalish) [reply bindings/c/meshIOc.cpp:55]: Thanks, and congrats! My first daughter was also a month early - had to spend a week in the NICU, which wasn't much fun. But she's fine and it's hard to believe she's ten now. Good job on getting out of grad school - want to share your dissertation? Enjoy your family time, and let me know if you want a Google referral.

File: src/manifold.cpp [hunk 2]
```diff
@@ -302,7 +301,7 @@ Manifold::Manifold(const MeshGL64& meshGL64)
  * front/back side. Each channel must be >= 3 and < numProp, and all original
```
Comment (pca006132) [inline src/manifold.cpp:301]: update documentation?

File: test/smooth_test.cpp [hunk 3]
```diff
@@ -268,18 +268,6 @@ TEST(Smooth, Csaszar) {
...
     options.mat.roughness = 0.1;
-
-    options.mat.vertColor.resize(csaszar.NumVert());
-    const vec4 blue(0, 0, 1, 1);
-    const vec4 yellow(1, 1, 0, 1);
-    for (size_t tri = 0; tri < csaszar.NumTri(); ++tri) {
-      for (int i : {0, 1, 2}) {
-        const vec3& uvw = {0.5, 0.5, 0.0};
-        const double alpha = std::min(uvw[0], std::min(uvw[1], uvw[2]));
-        options.mat.vertColor[out.triVerts[3 * tri + i]] =
```
Comment (pca006132) [inline test/smooth_test.cpp:279]: do we have color for this now?

**Review by pca006132 — APPROVED** [review]

File: test/smooth_test.cpp (hunk 3 above)
Comment (elalish) [reply test/smooth_test.cpp:279]: No, but it also wasn't working before - I realized this was leftover code that wasn't really doing much.

**Pull request merged**